src/packages/client/src/generation/TSClient/helpers.ts

```diff
@@ -188,61 +30,44 @@ export function getMethodJSDoc(
 ): string {
   return wrapComment(getMethodJSDocBody(action, mapping, model))
 }
-
+export function getGenericMethod(name: string, actionName: DMMF.ModelAction) {
+  if (actionName === 'count') return ''
+  if (actionName === 'aggregate')
+    return `<T extends ${getAggregateArgsName(name)}>`
+  return `<T extends ${getModelArgName(name, actionName)}>`
+}
+export function getArgs(name: string, actionName: DMMF.ModelAction) {
+  if (actionName === 'count')
```

Comment:
Let's please always use `if () {}` syntax with the curly braces. That's a convention we have all around the codebase for readability.